docs/native-modules-android.md

```diff
@@ -292,9 +412,9 @@ CalendarModule.createCalendarEvent('foo', 'bar');
 
 ### Argument Types
 
-When a native module method is invoked in JavaScript, React Native converts the arguments from JS objects to their Java object analogues. So for example, if your Java Native Module method accepts a double, in JS you need to call the method with a number. React Native will handle the conversion for you. Below is a list of the argument types supported for native module methods and the JavaScript equivalents they map to.
+When a native module method is invoked in JavaScript, React Native converts the arguments from JS objects to their Java/Kotlin object analogues. So for example, if your Java Native Module method accepts a double, in JS you need to call the method with a number. React Native will handle the conversion for you. Below is a list of the argument types supported for native module methods and the JavaScript equivalents they map to.
 
-| Java          | JavaScript |
+| Java/Kotlin   | JavaScript |
 | ------------- | ---------- |
 | Boolean       | ?boolean   |
 | boolean       | boolean    |
```

Comment:
Kotlin doesn't primitive types (e.g. `boolean`). 

Maybe `boolean` and `double` in this table should be exported to a separate Java only table?

Reply:
Same for `int` and `float` below